testing/testcafe/tests/dataGrid/validation.ts

```diff
@@ -0,0 +1,45 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import DataGrid from '../../model/dataGrid';
+import { getData } from './helpers/generateDataSourceData';
+
+fixture.disablePageReloads`Validation`
+  .page(url(__dirname, '../container.html'));
+
+test('Validation popup screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .maximizeWindow()
+    .click(dataGrid.getDataCell(0, 0).element)
+    .pressKey('ctrl+a backspace enter')
+    // act
+    .expect(await takeScreenshot('validation-popup', dataGrid.element))
+    .ok()
+    // assert
+    .expect(Selector('.dx-datagrid-revert-tooltip').exists)
+    .ok()
+    .expect(Selector('.dx-invalid-message.dx-invalid-message-always.dx-datagrid-invalid-message').exists)
```

Comment:
Let's not use ```Selector``` in our tests and add these to the DataGrid page object model instead